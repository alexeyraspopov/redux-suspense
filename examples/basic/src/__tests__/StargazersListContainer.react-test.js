import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import configureStore from '../configureStore';
import { Provider } from 'react-redux';
import StargazerListContainer from '../StargazerListContainer.react';

test('precached data render', () => {
  let state = {
    stargazers: {
      'alexeyraspopov/react-coroutine': [
        {
          login: 'alexeyraspopov',
          html_url: 'https://github.com/alexeyraspopov',
        },
        {
          login: 'istarkov',
          html_url: 'https://github.com/istarkov',
        },
        {
          login: 'terrysahaidak',
          html_url: 'https://github.com/terrysahaidak',
        },
        {
          login: 'RReverser',
          html_url: 'https://github.com/RReverser',
        },
      ],
    },
  };
  let store = configureStore(state);
  let output = ReactTestRenderer.create(
    <Provider store={store}>
      <StargazerListContainer repository="alexeyraspopov/react-coroutine" />
    </Provider>
  );
  expect(output).toMatchSnapshot();
});
