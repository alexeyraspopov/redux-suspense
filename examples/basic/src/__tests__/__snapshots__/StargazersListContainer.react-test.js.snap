// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<ul>
  <li>
    <a
      href="https://github.com/alexeyraspopov"
    >
      alexeyraspopov
    </a>
  </li>
  <li>
    <a
      href="https://github.com/istarkov"
    >
      istarkov
    </a>
  </li>
  <li>
    <a
      href="https://github.com/terrysahaidak"
    >
      terrysahaidak
    </a>
  </li>
  <li>
    <a
      href="https://github.com/RReverser"
    >
      RReverser
    </a>
  </li>
</ul>
`;
